examples/data-sources/dbt_cloud_privatelink_endpoint/data-source.tf

```diff
@@ -0,0 +1,13 @@
+data "dbt_cloud_privatelink_endpoint" "test_with_name" {
+  name = "My Endpoint Name"
+}
+
+data "dbt_cloud_privatelink_endpoint" "test_with_url" {
+  private_link_endpoint_url = "abc.privatelink.def.com"
+
```

Comment:
Nit: blank line 😅

Reply:
I have an empty line at the end locally, not sure why it didn't get pushed